components/main_menu/main_menu.test.jsx

```diff
@@ -163,6 +163,20 @@ describe('components/Menu', () => {
         expect(wrapper.find('#leaveTeam').props().show).toEqual(true);
     });
 
+    test('should show subscribe now button when in trial period', () => {
+        const props = {...defaultProps, isCloud: true, isFreeTrial: true};
+        const wrapper = getMainMenuWrapper(props);
+
+        expect(wrapper.find('UpgradeLink')).toHaveLength(1);
+    });
+
+    test('should hide the subscribe now button when NOT in trial period', () => {
```

Comment:
Can we add one more that it should not show the button if cloud is false ?

Reply:
done!